app/invocation/invocation_action_card.tsx

```diff
@@ -217,6 +221,10 @@ export default class InvocationActionCardComponent extends React.Component<Props
       return;
     }
     const digest = parseActionDigest(digestParam);
+    if (!digest) {
```

Comment:
nit: if you pass `digestParam ?? ""` to this func then you can remove the redundant check on line 219

Reply:
Done